CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## v0.3.1
+
+Conduit 0.3.1 improves Conduit's resilience and transparency.
+
+* Proxy (data plane)
+  * The proxy now makes fewer changes to requests and responses being proxied. In particular, requests and responses without bodies or with empty bodies are better supported.
```

Comment:
The rest of this file is wrapped at 100 characters -- can I bug you to do that for these long bullet points as well?

Reply:
> The rest of this file is wrapped at 100 characters -- can I bug you to do that for these long bullet points as well?

Done. (My editor wasn't 100 characters wide when I was editing this originally, so I didn't realize things were wrapped to 100 characters.)